pages/common/gh-pr-create.md

```diff
@@ -0,0 +1,24 @@
+# gh pr create
+
+> Manage GitHub pull requests from the command-line.
+> More information: <https://cli.github.com/manual/gh_pr_create>.
+
+- Interactively create a pull request:
+
+`gh pr create`
+
+- Create a pull request, determining the title and description from the commit messages of the current branch:
+
+`gh pr create --fill`
+
+- Create a draft pull request:
+
+`gh pr create --draft`
+
+- Create a pull request, specifying the base branch, title, and description:
+
+`gh pr create -B {{base_branch}} -t "{{pull_request_title}}" -b "{{pull_request_description}}"`
+
+- Start opening a pull request in the browser:
```

Comment:
How about ?
```suggestion
- Create a pull request and open it in the browser:
```

Reply:
Because it doesn't actually do that — it basically opens https://github.com/tldr-pages/tldr/compare only with your branch, so that you can configure everything in the UI.

Reply:
`Open a pull request`

Reply:
What's the problem with the original description? It reads better than any other suggested alteration here?